client/src/main/java/org/opensearch/ml/client/MachineLearningClient.java

```diff
@@ -24,6 +24,7 @@ public interface MachineLearningClient {
 
     /**
      * Do prediction machine learning job
+     * For additional info on Predict, refer: https://opensearch.org/docs/latest/ml-commons-plugin/api/#predict
```

Comment:
Seems this should be removed, otherwise will conflict with the earlier on from Bhavana.

Reply:
Yes, rebased code